src/components/password-modal/components/password-pin/password-pin.tsx

```diff
@@ -267,23 +267,27 @@ export class PasswordPinComponent extends React.Component<
 
     public renderFooterRow = () => {
         const styles = this.props.styles;
+        const icon =
+            Platform.OS === 'ios' && this.state.biometryType
```

Comment:
this condition is way too complicated :))
i think it would be more simple to do this:
```
const icon = Platform.OS === 'ios' && this.state.biometryType === 'FaceID' ? 'face-id' : 'touch-id';
```

Reply:
Yes indeed :)) Fixed